elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1330,7 +1366,12 @@ protected void nullValue(String fieldName, PersistentResource oldValue) {
         }
         String inverseField = getInverseRelationField(fieldName);
         if (!inverseField.isEmpty()) {
-            oldValue.checkFieldAwareDeferPermissions(UpdatePermission.class, inverseField, null, getObject());
+            if (isNewlyCreated()) {
```

Comment:
Should we check `oldValue.isNewlyCreated()` instead of `this.isNewlyCreated()` since we are calling `oldValue.checkFieldAwareDeferPermissions` below?

Reply:
Yes.